scripts/exgdas_global_atmos_analysis_run.sh

```diff
@@ -126,7 +126,7 @@ for fieldset in $fieldsets; do
 done
 
 # CRTM coeffs
-$NLN $FV3JEDI_FIX/crtm/2.3.0_jedi $DATA/crtm
+${NLN} "${FV3JEDI_FIX}"/crtm/2.3.0 "${DATA}"/crtm
```

Comment:
Use quotes around each entire path instead of just single variables.

Reply:
Requested change made and committed at 5ecf08d0.